quantum/matrix.c

```diff
@@ -62,7 +62,8 @@ static bool read_cols_on_row(matrix_row_t current_matrix[], uint8_t current_row)
     return (last_row_value != current_matrix[current_row]);
 }
 
-#elif (DIODE_DIRECTION == COL2ROW)
+#elif defined(DIODE_DIRECTION)
+#    if DIODE_DIRECTION == COL2ROW
```

Comment:
Probably doesn't matter, but all the other comparable statements in this PR use parentheses.

```suggestion
#    if (DIODE_DIRECTION == COL2ROW)
```

Reply:
I thought I got all of them... 😑